source/include/FreeRTOS_IP.h

```diff
@@ -97,22 +97,29 @@ extern uint32_t ulApplicationGetNextSequenceNumber( uint32_t ulSourceAddress,
  * pointer back to the network buffer.  Should be a multiple of 8 to ensure 8 byte
  * alignment is maintained on architectures that require it.
  *
- * In order to get a 32-bit alignment of network packets, an offset of 2 bytes
- * would be desirable, as defined by ipconfigPACKET_FILLER_SIZE.  So the malloc'd
+ * In order to get a 32-bit or 64-bit alignment of network packets, an offset of 2 bytes
+ * is ideal as defined by ipconfigPACKET_FILLER_SIZE. So the malloc'd
  * buffer will have the following contents:
- *  uint32_t pointer;   // word-aligned
- *  uchar_8 filler[6];
- *  << ETH-header >>    // half-word-aligned
- *  uchar_8 dest[6];    // start of pucEthernetBuffer
- *  uchar_8 dest[6];
- *  uchar16_t type;
- *  << IP-header >>     // word-aligned
+ *
+ * +---------+-----------+---------+
+ * | Offset  | Alignment | Length  |
+ * | 32 | 64 | 32  | 64  | 32 | 64 |
+ * |----|----|-----|-----|----|----|
+ * | 0  | 0  | 4   | 8   | 4  | 8  | uchar_8 * pointer;     // Points to the 'NetworkBufferDescriptor_t'.
+ * | 4  | 8  | 4   | 8   | 2  | 2  | uchar_8   filler[];    // To give the +2 byte offset.
+ * | 6  | 10 | 4+2 | 8+2 | 6  | 6  | uchar_8   dest_mac[6]; // Destination address.
```

Comment:
In the case of 32 bit platforms the `ipBUFFER_PADDING` is `( 8U + ipconfigPACKET_FILLER_SIZE )` by default. And `ipconfigPACKET_FILLER_SIZE` is 2 by default. In that case shouldn't the `dest_mac` start at an offset of 10 bytes from the start of the network buffer. Similarly 14 bytes for 64 bit platforms?

Reply:
There were indeed some typos in the table in the comments. Hopefully all correct now.